AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -467,10 +467,8 @@ - (void)setPlayerState:(ASVideoNodePlayerState)playerState
 
 - (void)setAssetURL:(NSURL *)assetURL
 {
-  ASDN::MutexLocker l(__instanceLock__);
-
   if (ASObjectIsEqual(assetURL, self.assetURL) == NO) {
```

Comment:
Yes, there is a small chance that it happens.

I've just discovered that these properties are always set on main. I'm asking people to see if the same restriction applies to their getters. If it's also the case, we can replace these lockings with main thread assertions!